lab2.js

```diff
@@ -44,14 +44,21 @@ function assert(expression, failureMessage) {
     fourth hour     |    4/hour        |        4         |
 
  TODO: First, make a constructor function, called Blob, that makes blobs.
-
+  
  TODO: Next, create an instance of Blob named blob.
 
  TODO: Then, use a loop to calculate how long it took the blob to finish
  with Dowington. */
 
+function Blob() {}
+
+var blob = new Blob();
+
+for (var i = 0; i < population.length; i++)
 
-var hoursSpentInDowington; // TODO: assign me the value of the
+
+var hoursSpentInDowington = var hoursSpentInDowington = blob.hoursToOoze(1000, 1);
```

Comment:
this line of code is throwing an error - you have var hoursSpentInDowington twice. Be sure to run your file with node lab2.js (in the terminal) as you work, so you can catch errors like this.